projects/step-frontend/src/lib/modules/execution/services/tree-search-description.token.ts

```diff
@@ -0,0 +1,7 @@
+import { InjectionToken } from '@angular/core';
+
+export const TREE_SEARCH_DESCRIPTION = new InjectionToken<string>('Tree search description', {
+  providedIn: 'root',
+  factory: () =>
+    'The search is performed on the raw-data. This means configurations are found by there ID instead the rendered labels shown in the tree. This may lead to false-positives or false-negative search results.',
```

Comment:
```suggestion
    'Search is performed on the raw data. As a result, configurations are matched by their IDs rather than the labels shown in the tree. This may lead to search results that differ from what is displayed, including occasional false positives or missed matches.',
```